### PR TITLE
Enable the Ray dashboard by default

### PR DESCRIPTION
**Pull request by ericl**

### CODE REVIEW

File: python/ray/services.py [hunk 1]
```diff
@@ -1034,21 +1035,11 @@ def start_dashboard(host,
...
-        host if host == "127.0.0.1" else get_node_ip_address(), port)
-    print("\n" + "=" * 70)
-    print("View the dashboard at {}.".format(dashboard_url))
-    if host == "127.0.0.1":
-        note = (
-            "Note: If Ray is running on a remote node, you will need to set "
-            "up an SSH tunnel with local port forwarding in order to access "
-            "the dashboard in your browser, e.g. by running "
-            "'ssh -L {}:{}:{} <username>@<host>'. Alternatively, you can set "
-            "webui_host=\"0.0.0.0\" in the call to ray.init() to allow direct "
```
Comment (ericl) [inline python/ray/services.py]: This should go in the docs

File: python/ray/worker.py [hunk 2]
```diff
@@ -83,6 +83,11 @@
 # entry/init points.
 logger = logging.getLogger(__name__)
 
+try:
+    import aiohttp
+except ImportError:
+    aiohttp = None
```
[the remark this answers is not in the document]
Comment (ericl) [reply python/ray/worker.py]: Yep, it's warned below.

[the remark this answers is not in the document]
Comment (ericl) [reply python/ray/worker.py]: Yep, it's printed in node.py

**Review by mitchellstern — COMMENTED** [review]
The changes made here look good, but if we want to update the default host from 127.0.0.1 to localhost, then that should be applied everywhere, including:

- [X] `python/ray/dashboard/dashboard.py` (for the `--host` option)
- [x] `python/ray/parameter.py` (for both the class docstring and the `__init__()` default parameter in the `RayParams` class)
- [x] `python/ray/scripts/scripts.py` (for the `--webui-host` option)
- [x] `python/ray/tests/test_webui.py` (I'm not sure why this hasn't shown up in the tests, but it will currently fail if localhost is used as the default host)
- [x] `python/ray/worker.py` (for both the default parameter and the docstring in `init()`)

Please do a `grep` for 127.0.0.1 and/or see the changes from #5888 to make sure that it's replaced by localhost in all the relevant places, thanks!

File: python/ray/services.py (hunk 1 above)
Comment (mitchellstern) [reply python/ray/services.py]: Ok, I'll follow up on that after this PR gets merged.

**Comment by ericl** [comment]
Done. Note that in certain cases it gets resolved back to 127.0.0.1 so no need to change.

**Comment by ericl** [comment]
I also removed some areas which required localhost or 0.0.0.0, in general host should be settable to any value.

**Comment by mitchellstern** [comment]
Can we fix `python/ray/tests/test_webui.py` by updating the regex to account for the new URL format, i.e. by dropping the http:// portion and by allowing for localhost? Maybe something like the following?
```
assert re.match(r"^(localhost|\d+\.\d+\.\d+\.\d+):8080$", webui_url)
```
(Note, I haven't tested the above yet with this PR.)

**Comment by ericl** [comment]
Fixed

**Review by mitchellstern — APPROVED** [review]

**Pull request merged**